lib/src/tenant_token/exceptions.dart

```diff
@@ -1,7 +1,13 @@
 part of tenant_token;
 
-class ExpiredSignatureException implements Exception {}
+class ExpiredSignatureException implements Exception {
+  const ExpiredSignatureException();
+}
 
-class NotUTCException implements Exception {}
+class NotUTCException implements Exception {
+  const NotUTCException();
```

Comment:
`const` is merely an optimization keyword, meant to inform the compiler that this class can be created in compile time.
and no it's not a singleton, so `const NotUTCException() != const NotUTCException()` and it would produce a lint